public/index.scss

```diff
@@ -31,27 +20,31 @@
     }
   }
 
-  .llm-chat-header-shortcut {
-    display: flex;
-    align-items: center;
-    padding-right: 6px;
+  @media (max-width: 1200px) {
+    &.in-singleline-header{
+      display: none;
+    }
+  }
```

Comment:
Nit: seems we can leverage https://oui.opensearch.org/1.19/#/utilities/css-utility-classes#responsive .